integration_test/common/testdata/integration-metadata_oneof_field_expected-metric/golden_error

```diff
@@ -0,0 +1 @@
+Key: 'IntegrationMetadata.ExpectedMetrics' Error:Field validation for 'ExpectedMetrics' failed on the 'oneof_field' tag
```

Comment:
I'm confused — shouldn't `oneof` [produce a more sensible error message](https://github.com/go-playground/validator/blob/master/translations/en/en.go#L1279-L1291) like `monitored_resource must be one of ["gce_instance"]`? Where is this one coming from?

Reply:
Discussed offline. These default messages require one to explicitly register the supplied translations, which is nontrivial. We will likely add a switch similar to the [one in the config validator](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/config.go#L90-L129), but in a follow-up change.